source/content/guides/drupal/drupal-hosted/07-code.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: Migrate a Drupal 8 Site to Drupal 9
+title: Migrate a Drupal 8 Site to Drupal
```

Comment:
@stovak , contains multiple references to D8. Should it say “prior version” instead?  See preview link:https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-hosted/code